accessibilitySprint2015.html

```diff
@@ -183,48 +206,22 @@
         <section class="agenda" id="agenda">
             <h1>Agenda</h1>
             <p>To be announced...</p>
-<!--
-            <table class="agenda-table">
-                <tr><td>9:00 - 10:00</td><td>Welcome</td></tr>
-                <tr><td>10:00 - 11:00</td><td>Stuff</td></tr>
-                <tr><td>11:00 - 12:00</td><td>other stuff</td></tr>
-                <tr><td>12:00 - 1:00</td><td>more stuff</td></tr>
-                <tr><td>1:00 - 2:00</td><td>Lunch!</td></tr>
-                <tr><td>2:00 - 3:00</td><td>post-lunch walk</td></tr>
-                <tr><td>3:00 - 4:00</td><td>more stuff</td></tr>
-                <tr><td>4:00 - 5:00</td><td>different stuff</td></tr>
-                <tr><td>5:00 - 6:00</td><td>non-stuff</td></tr>
-                <tr><td>6:00 - 7:00</td><td>dinner</td></tr>
-                <tr><td>7:00 - 8:00</td><td>drinking!</td></tr>
-                <tr><td>8:00 - 9:00</td><td>unmentionable</td></tr>
-            </table>
--->
         </section>
 
         <section class="partners" id="partners">
             <h1>Partners &amp; Attendees</h1>
-            <div class="fl-col-flex3">
-                <ul class="fl-col">
-                    <li>Inclusive Design Research Centre, OCAD University</li>
-                    <li>Mozilla</li>
-                    <li>PhET</li>
-                </ul>
-                <ul class="fl-col">
-                    <li>WGBH National Center for Accessible Media</li>
-                    <li>MIT</li>
-                </ul>
-                <ul class="fl-col">
-                    <li>Raising the Floor</li>
-                    <li>George Brown College</li>
-                    <li>... and many more!</li>
-                </ul>
+            <div>
+                Over 60 global partners and institutions will be attending, RSVP today!
```

Comment:
I think this needs to be something like "Over 60 attendees from our partners across the world." Also the registration might be closed now, so we can drop the RSVP part.